### PR TITLE
refactor: change root options

### PR DESCRIPTION
**Pull request by jpower432**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: pkg/bundle/doc.go [hunk 1]
```diff
@@ -1,2 +1,2 @@
-// Package bundle contains tools for managing container images for internet-disconnected systems.\.
+// Package bundle contains tools for preparing imagesets for archiving.\.
```
Comment (estroz) [inline pkg/bundle/doc.go:1]:
```suggestion
// Package bundle contains tools for preparing imagesets for archiving.
```

File: pkg/cli/mirror/additional.go [hunk 2]
```diff
@@ -11,18 +11,17 @@ import (
 	"github.com/openshift/oc/pkg/cli/image/mirror"
 	"github.com/sirupsen/logrus"
 
-	"github.com/RedHatGov/bundle/pkg/cli"
```
Comment (estroz) [inline pkg/cli/mirror/additional.go:14]: Is the plan to update the module path to `github.com/openshift/oc-mirror` in a separate PR?

File: pkg/cli/mirror/options.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	SkipCleanup      bool
+	FilterOptions    imagemanifest.FilterOptions
+
+	BuildxPlatforms []string
+}
+
+func (o *MirrorOptions) BindFlags(fs *pflag.FlagSet) {
+	fs.StringVarP(&o.ConfigPath, "config", "c", o.ConfigPath, "Path to imageset configuration file")
+	fs.BoolVar(&o.SkipImagePin, "skip-image-pin", o.SkipImagePin, "Do not replace image tags with digest pins in operator catalogs")
+	fs.StringVar(&o.From, "from", o.From, "The archive file path.")
```
Comment (estroz) [inline pkg/cli/mirror/options.go:30]: Did we decide to improve the help text for this flag?

File: pkg/cli/mirror/options.go [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+	DestSkipTLS      bool
+	SkipVerification bool
+	SkipCleanup      bool
+	FilterOptions    imagemanifest.FilterOptions
+
+	BuildxPlatforms []string
+}
+
+func (o *MirrorOptions) BindFlags(fs *pflag.FlagSet) {
+	fs.StringVarP(&o.ConfigPath, "config", "c", o.ConfigPath, "Path to imageset configuration file")
```
Comment (estroz) [inline pkg/cli/mirror/options.go:28]: Some flag help texts start with upper case letters, some with lower case. Let's choose one (I prefer upper).

**Review by estroz — CHANGES_REQUESTED** [review]

File: pkg/cli/mirror/options.go (hunk 4 above)
Comment (jpower432) [reply pkg/cli/mirror/options.go:28]: +1 to upper. I will make this change.

File: pkg/cli/mirror/options.go (hunk 3 above)
Comment (jpower432) [reply pkg/cli/mirror/options.go:30]: @estroz We did on a previous PR. Was actually looking for ideas on exactly what we want here. I still agree that it needs to be updated. I can try to come up with something and update the PR.

File: pkg/cli/mirror/additional.go (hunk 2 above)
Comment (jpower432) [reply pkg/cli/mirror/additional.go:14]: Yea I will do this in a separate PR once this PR goes through.

**Review by estroz — COMMENTED** [review]
/lgtm

**Review by estroz — APPROVED** [review]

**Pull request merged**